go/pkg/client/cas.go

```diff
@@ -360,6 +371,7 @@ func (c *Client) uploadNonUnified(ctx context.Context, data ...*uploadinfo.Entry
 						return err
 					}
 					bchMap[dg] = data
+					totalBytesTransferred += int64(len(data))
```

Comment:
That should be protected somehow (different goroutines writing to same variable).
Please run test with features=race to verify.

Reply:
nit: using sync/atomic AddUint64 is shorter.

Reply:
Done